_data/simple-icons.json

```diff
@@ -5717,7 +5718,7 @@
         {
             "title": "Hackaday",
             "hex": "1A1A1A",
-            "source": "https://hackaday.com"
+            "source": "https://hackaday.com/wp-content/themes/hackaday-2/img/logo.svg"
```

Comment:
I prefer to keep the domain only. The link to the WordPress theme asset is subject to change.

Reply:
The SVG version doesn't exist on the front page. So in that case should we still link to front page?

Reply:
Please let us know how you find this resource link.

Reply:
It _is_ available in the header of the site as the 'mask icon' link. I'd say let's just keep the homepage URL.

Reply:
Reverted this change in another commit